src/ui/DynamicFacet/DynamicFacet.ts

```diff
@@ -135,8 +135,8 @@ export class DynamicFacet extends Component implements IDynamicFacet {
      *
      * @examples score
      */
-    sortCriteria: ComponentOptions.buildStringOption({
-      postProcessing: value => (isFacetSortCriteria(value) ? (value as FacetSortCriteria) : undefined),
+    sortCriteria: <FacetSortCriteria>ComponentOptions.buildStringOption({
```

Comment:
I'm not sure if it makes a difference for TypeDoc, but it's possible to pass a generic type to `buildStringOption`. e.g.
```
ComponentOptions.buildStringOption<FacetSortCriteria>(...)
```
That said, I'm curious to know if TypeDoc still sets the `constrainedValues` correctly for the sortCriteria option when using a cast.